### ISSUE TITLE
Can't sign in to Nook for PC

### ISSUE THREAD

**Issue by MargretRose** [apprenticeharper/DeDRM_tools#2309]
I downloaded the Nook app from Apprentice Alf's blog and installed it on my new Win 11 computer with no difficulty. But when I tried to sign in I got an error message: "We've encountered a network error - please try again. (Sign in: 1017)"

I have Edge set to allow Internet Explorer emulation. "Use TLS 1.2" is checked in Advanced Internet Options. I've even set Edge as my default web browser, but I'm still getting the error message.

Any ideas how to sign in to Nook in Win 11?

**Comment by ElleKayEm** [comment]
Those old apps were cut off years ago.  The Microsoft store Nook app was removed.  So if you don't already have it, you can't install it..  Here are write ups of how to get your key and books from an older Nook Android app in an emulator, but I don't know if this method works anymore either.
https://www.aricrenzo.com/2019-12-13-Liberate-Your-Nook-Ebooks/
https://kmhilton.wordpress.com/2021/05/25/dedrm-your-nookbooks-for-dummies/

**Comment by MargretRose** [comment]
Thank you. I've been up all night working on this, with disappointing results. I have the virtual phone set up. I have a Nook reader installed on it and was able to sign in. What I have **not** been able to do so far are to set up a virtual SD card, nor to find any books at all in my Nook library - the app recognizes me but thinks I haven't bought any books.

The reason I haven't been able to set up a virtual SD card is that it wants to reformat part of my hard disk, erasing anything that's already there, and it didn't offer to allow me to tell it where to set the virtual SD card up, which makes me squeamish. As for the Nook app thinking I haven't bought any books, I'm clueless.

When I followed the instructions on DeDRM your Nookbooks for Dummies, everything worked, right up to step 23, at which point I got an error message stating "Select-Object : A positional parameter cannot be found that accepts argument
'from'." I suspect this is because the guide was written with Win 10 in mind, and I'm using Win 11.

When I followed the instructions on Liberate Your Nook Ebooks, sqlite 3 gave me the error message "Parse error: no such table: cc_hash_data" I suspect this is because I neither have a virtual drive set up nor any books showing up in the library.

In other words, so far I can retrieve neither my key nor my books. I'm starting to wonder about creating a bootable Linux flash drive.

**Comment by ElleKayEm** [comment]
I'm afraid I can't be much help.  I liberated my nookbooks before this was necessary and have avoided them since.  I don't know if the problem may be that BN isn't allowing access to the older Android app or what.  You'd think it wouldn't let you log in or would say something about needing to update.
